bin/land.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const childProcess = require('child_process');
+const process = require('process');
+const https = require('https');
+
+const landedBranch = process.argv.slice(2);
+
+const options = {
+  headers: {
+    'User-Agent': 'request',
+  },
+  host: 'api.github.com',
+};
+
+const getPRURL = () => {
+  const configArgs = ['config', '--get', 'remote.origin.url'];
+  const gitConfig = childProcess.spawn('git', configArgs);
+
+  gitConfig.stdout.on('data', data => {
+    const correctData = data.toString('utf8');
+    options.path = `/search/issues?q=repo:${correctData.slice(
```

Comment:
Could you extract this `slice` in a variable to avoid multiline template like this please?